docs/src/main/asciidoc/user-guide/topics/import_files_to_panel.adoc

```diff
@@ -0,0 +1,284 @@
+<d:SchemaSet xmlns:d="http://atlasmap.io/xml/schemaset/v2" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <xsd:schema targetNamespace="http://syndesis.io/v1/swagger-connector-template/request" elementFormDefault="qualified">
+      <xsd:element name="request">
+        <xsd:complexType>
+          <xsd:sequence>
+            <xsd:element name="body">
+              <xsd:complexType>
+                <xsd:sequence>
+                  <xsd:element ref="Pet" />
+                </xsd:sequence>
+              </xsd:complexType>
+            </xsd:element>
+          </xsd:sequence>
+        </xsd:complexType>
+      </xsd:element>
+    </xsd:schema>
+    <d:AdditionalSchemas>
+      <xsd:schema>
+        <xsd:element name="Pet">
+          <xsd:complexType>
+            <xsd:sequence>
+              <xsd:element name="id" type="xsd:decimal" />
+              <xsd:element name="Category">
+                <xsd:complexType>
+                  <xsd:sequence>
+                    <xsd:element name="id" type="xsd:decimal" />
+                    <xsd:element name="name" type="xsd:string" />
+                  </xsd:sequence>
+                </xsd:complexType>
+              </xsd:element>
+              <xsd:element name="name" type="xsd:string" />
+              <xsd:element name="photoUrl">
+                <xsd:complexType>
+                  <xsd:sequence>
+                    <xsd:element name="photoUrl" type="xsd:string" maxOccurs="unbounded" minOccurs="0" />
+                  </xsd:sequence>
+                </xsd:complexType>
+              </xsd:element>
+              <xsd:element name="tag">
+                <xsd:complexType>
+                  <xsd:sequence>
+                    <xsd:element name="Tag" maxOccurs="unbounded" minOccurs="0">
+                      <xsd:complexType>
+                        <xsd:sequence>
+                          <xsd:element name="id" type="xsd:decimal" />
+                          <xsd:element name="name" type="xsd:string" />
+                        </xsd:sequence>
+                      </xsd:complexType>
+                    </xsd:element>
+                  </xsd:sequence>
+                </xsd:complexType>
+              </xsd:element>
+              <xsd:element name="status" type="xsd:string" />
+            </xsd:sequence>
+          </xsd:complexType>
+        </xsd:element>
+      </xsd:schema>
+    </d:AdditionalSchemas>
+  </d:SchemaSet>
+```
+As was done in the previous JSON file import, select the import icon this time in the *Target* panel.
+
+image:ImportXML1.png[Import an XML File to Target Panel]
+
+An *Open File* dialog appears.  Select the location for XMLSchema.xml.
+The fields now appear in the *Target* panel as you've defined them in the imported XML schema.  The fields
+are expanded to show more detail.
+
+image:ImportXML2.png[Import of an XML File to Target Panel Complete]
+
+Instance files are imported in precisely the same manner.  These files define a separate namespace
```

Comment:
Instead: You import instance files in the same way. Instance files define a separate namespace, which also defines a few special attributes. For example: